required_files/iGame.guide

```diff
@@ -1,325 +1,360 @@
 @DATABASE iGame.guide
 @$VER: iGame.guide 2.0b6 (04.09.2020)
 @INDEX "Index"
+@wordwrap
 
 @NODE "Main" "iGame documentation"
-
-
-			   iGame
-	=============================================
-		� 2005-2020 Emmanuel Vasilakis
-
-     @{"�Introduction���� " LINK "INTR" 0} - About iGame
-     @{"�Building iGame�� " LINK "BUIL" 0} - Building iGame
-     @{"�Requirements���� " LINK "REQS" 0} - System requirements
-
-     @{"�Installation���� " LINK "INST" 0} - Installing iGame
-     @{" Usage            " LINK "USAG" 0} - Using iGame
-
-     @{"�Todo      ������ " LINK "TODO" 0} - Todo stuff
-     @{" Known bugs       " LINK "BUGS" 0} - Known bugs
-     @{"�Author���������� " LINK "INFO" 0} - Author contact info
-     @{"�History  ������� " LINK "HIST" 0} - Version history
+@{jcenter}
+iGame
+=============================================
+(c) 2005-2020 Emmanuel Vasilakis
+@{jleft}
+@{lindent 3}
+@{" Introduction    " LINK "INTR" 0} - About iGame
+@{" History         " LINK "HIST" 0} - History of iGame
+@{" Building iGame  " LINK "BUIL" 0} - Building iGame
+@{" Requirements    " LINK "REQS" 0} - System requirements
+
+@{" Installation    " LINK "INST" 0} - How to install iGame
+@{" Usage           " LINK "USAG" 0} - Using iGame
+
+@{" Todo & Bugs     " LINK "TODO" 0} - Todo and known bugs
+@{" Author          " LINK "INFO" 0} - Author contact info
+@{" Changelog       " LINK "CHNG" 0} - Version changelog
 
 @ENDNODE
 @NODE "INTR" "Introduction"
+@{lindent 3}
+iGame is a frontend application for your games and demos collection. It runs on AmigaOS 2.04 and above, AmigaOS 4 and MorphOS.
 
-iGame is a program that provides a front-end to games installed with whdload
-on your machine.
+@{b}Features@{ub}
 
-It can "discover" your games on pre-defined repositories and create a small
-database. You can then categorize each game according to it's genre, provide
-a small image to be displayed when the game is selected, and quickly find the
-one you're looking for using the Filter gadget.
+ - Multiple WHDLoad slaves repositories on hard disk partitions
+ - On demand scanning in repositories for installed WHDLoad slaves (games, demos etc.)
+ - Use games' tooltypes on run
+ - Shows game screenshot (screenshot window can be altered through tooltypes/settings, uses datatypes to load foreign formats)
+ - Categorization of the games and filtering
+ - Manual addition of non-WHDLoad games, demos etc.
+ - Simple statistics
+ - Find-as-you-type search filter
 
-But not just whdload games. You can add and launch any type of executable, though
-it mostly makes sense if it's a game ;-)
+iGame can "discover" your games on pre-defined repositories and create a small database. You can then categorize each game according to it's genre, provide a small screenshot image to be displayed when the game is selected and quickly find the one you're looking for, using the filter gadget.
```

Comment:
"it's" should be "its" (without apostrophe) since this use is not a contraction of "it is".